msteams-platform/get-started/glossary.md

```diff
@@ -262,6 +264,7 @@ Common terms and definitions used in Microsoft Teams developer documentation.
 | [Thread discussion](../tabs/design/tabs.md#thread-discussion) | A conversation posted on a channel or chat between users. <br>**See also** [Conversation](#c); [Channel](#c) |
```

Comment:
Line 122: That's more like a definition for Fluent UI components. So, I would say, Fluent UI is a front-end framework for building consistent, accessible, and visually cohesive user interfaces across Teams applications and beyond."